src/luxor_overrides.jl

```diff
@@ -46,13 +46,80 @@ function setopacity(opacity)
     Luxor.setopacity(current_opacity)
 end
 
+"""
+    fontsize(fsize)
+
+Same as `Luxor.fontsize`: Sets the current font size.
+
+# Example
+```
+fontsize(12)
+text("Hello World!")
+```
+
+# Arguments:
+- `fsize`: the new font size
+"""
 function fontsize(fsize)
     cs  = get_current_setting()
     cs.fontsize  = fsize
     Luxor.fontsize(fsize)
 end
 
+"""
+    get_fontsize(fsize)
+
+Same as `Luxor.get_fontsize` but works with every version of Luxor that is supported by Javis.
+
+# Example
+```
+fontsize(12)
+fsize = get_fontsize()
+text("Hello World! \$fsize")
+```
+
+# Returns
+- `Float64`: the current font size
+"""
 function get_fontsize()
     cs  = get_current_setting()
     return cs.fontsize
 end
+
+"""
+    scale(scl)
+
+Set the scale and multiply it with the current multiplier
+which is i.e. set by [`appear`](@ref) and [`disappear`](@ref).
+
+Normal behavior without any animation is the same as `Luxor.scale`.
+
+# Example
+```
+scale(0.5)
+circle(O, 20, :fill) # the radius would be 10 because of the scaling
+```
+
+# Arguments:
+- `scl`: the new default scale
+"""
+function scale(scl::Number)
+    scale(scl, scl)
+end
+
+"""
+    scale(scl_x, scl_y)
+
+Same as [`scale`](@ref) but the x scale and y scale can be changed independently.
+
+# Arguments:
+- `scl_x`: scale in x direction
+- `scl_y`: scale in y direction
+"""
```

Comment:
Could you give an example of this? This is an interesting functionality that I am not sure if I understand how it works.

Reply:
It just scales in two directions so it can stretch a square into a rectangle as an example.

Reply:
Oh! That is nifty! Could you give an example of this? Thanks!

Reply:
An example is inside the `test/animations.jl` named something like "squeezing" :smile: